exonum/src/events/network.rs

```diff
@@ -311,11 +312,6 @@ impl RequestHandler {
                         let fut = cancel_sender
                             .take()
                             .ok_or_else(|| other_error("shutdown twice"))
```

Comment:
Is it real scenario? I mean, with this change, cancel_handler will return `Ok` in any situation, won't it?

Reply:
It seems to be no way to do it twice.